### ISSUE TITLE
Sitemap fails to generate on first-time/clean build

### ISSUE THREAD

**Issue by blkeller** [wackerservices/SitemapAutogenerator#1]
If sitemap-autogenerator is installed and configured as part of an Ember project, but the project has never been built before, `sitemap.xml` does not get created and the build throws a warning:
```
{ Error: ENOENT: no such file or directory, open 'dist/sitemap.xml'
    at Error (native)
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'dist/sitemap.xml' }
```

I believe this is because the `dist/` directory does not yet exist at the time the add-on runs, but the add-on assumes that it does.  You can also reproduce the problem on a clean build, which usually involves deleting the entire `dist/` directory as a part of the cleaning process.  Subsequent builds are not affected because at that point `dist/` is already present.

**Comment by rupdegraff** [comment]
This issue has been addressed in commit 02add168bff041f7ba0aac0e60b28c5c2c5a3dd4 and the following `package.json` code replaces the previous method of calling `sitemap-autogenerator` from `ember-cli-build.js`

## Usage

  Add the following code to **package.json**: 
```js
"postbuild": "node -e \"require('./node_modules/sitemap-autogenerator/blueprints/sitemap-autogenerator/index').triggerSitemapBuilder('<YOUR SITE ROOT URL>')\"",
```

  Be sure to not have a trailing `/` after `<YOUR SITE ROOT URL>` or else you will get double `//` in your generated **sitemap.xml**
  
  `sitemap-autogenerator` will run at the end of each Ember build, which are run with: `npm build`

**Comment by blkeller** [comment]
Perfect!  Thank you.
